src/axom/mint/docs/sphinx/sections/getting_started.rst

```diff
@@ -216,6 +285,14 @@ with ID, ``cellIdx`` is stored at ``xc[ cellIIdx * NUM_COMPONENTS  ]`` and the
 y-component is stored at ``xc[ cellIdx * NUM_COMPONENTS + 1 ]``, where
 ``NUM_COMPONENTS=2``.
 
+In addition, the node-centered quantity, ``phi``, computed in :ref:`step4` is
+averaged and stored at the cell centered, by first summing all the nodal
```

Comment:
Typo: change "centered" to "centers".  Additionally, I recommend to break the sentence here, resulting in something like this:  "...at the cell centers.  The kernel first sums all the nodal contributions in ``hsum`` and then multiplies by..."

Reply:
Done.